src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/FilingsService.java

```diff
@@ -130,6 +140,7 @@ private boolean setSubmissionData(Map<String, Object> data, String overseasEntit
         }
         data.put(PRESENTER_FIELD, submissionDto.getPresenter());
         data.put(ENTITY_FIELD, submissionDto.getEntity());
+        data.put(ENTITY_NUMBER_FIELD, submissionDto.getEntityNumber());
```

Comment:
We will only add this for update. But not now so revert line 143. This data goes to the chips filing consumer don't want this for registration! Means you will need to find another way to pass back the entity number. 
Please have a look at: https://github.com/companieshouse/overseas-entities-api/pull/210
This gets the Dto and has conditional logic for update/registration.